src/components/ListTasks.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { FlatList } from 'react-native';
+import Task from './Task';
+
+const data = [
```

Comment:
As a recommendation you could include all the mocked data in a separate script to keep the code cleaner.